test/unit/test_sap_cli_gcts.py

```diff
@@ -597,6 +597,42 @@ def test_repo_set_url(self, fake_set_url):
         fake_set_url.assert_called_once_with(new_url)
 
 
+class TestqCTSUserGetCredentials(PatcherTestCase, ConsoleOutputTestCase):
+
+    def setUp(self):
+        super().setUp()
+        ConsoleOutputTestCase.setUp(self)
+
+        assert self.console is not None
+
+        self.patch_console(console=self.console)
+        self.fake_connection = None
+        self.api_url = 'https://api.github.com/v3/'
+        self.fake_get_credentials = self.patch('sap.rest.gcts.simple.get_user_api_token')
+        self.fake_get_credentials.return_value = [{'endpoint': self.api_url, 'type': 'token', 'state': 'false'}]
+
+    def get_credentials_cmd(self, *args, **kwargs):
+        return parse_args('user', 'get-credentials', *args, **kwargs)
+
+    def test_get_user_credentials_json(self):
+        output_format = 'JSON'
+
+        the_cmd = self.get_credentials_cmd('-f', output_format)
+        the_cmd.execute(self.fake_connection, the_cmd)
+
+        self.assertConsoleContents(self.console,
+                                   stdout=f"[{{'endpoint': '{self.api_url}', 'type': 'token', 'state': 'false'}}]\n")
+
+    def test_get_user_credentials_human(self):
+        output_format = 'HUMAN'
+
+        the_cmd = self.get_credentials_cmd('-f', output_format)
+        the_cmd.execute(self.fake_get_credentials, the_cmd)
+
+        self.assertConsoleContents(self.console,
+                                   stdout=f"Endpoint: {self.api_url}\nType: token\nState: false\n")
```

Comment:
Please, do use multiline strings so reviewrs do not need to assemble the output in their minds.